generated/python/grpc-google-iam-v1/setup.py

```diff
@@ -10,14 +10,14 @@
 from setuptools import setup, find_packages
 
 install_requires = [
-  'oauth2client>=1.4.11, <2.0.0',
-  'grpcio>=1.0.0, <2.0.0',
-  'googleapis-common-protos[grpc]>=1.3.4, <2.0.0'
+  'oauth2client>=3.0.0, <4.0.0dev',
```

Comment:
@jonparrott is currently questioning the tightness of these bounds. Let's see what his respond to https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2457#issuecomment-250319781 is.